src/utils/fetchClient.ts

```diff
@@ -33,7 +34,9 @@ function request<T>(
 
 export const client = {
   get: <T>(url: string) => request<T>(url),
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   post: <T>(url: string, data: any) => request<T>(url, 'POST', data),
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   patch: <T>(url: string, data: any) => request<T>(url, 'PATCH', data),
```

Comment:
Avoid using any type. In TypeScript, the any type is used as a last resort when we cannot declare a specific type. It's better to use more specific type for 'data' parameter or at least an unknown type.